contrib/debian/netdata.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=netdata real-time system monitoring
+After=network.target
+
+[Service]
+Type=simple
+EnvironmentFile=-/etc/default/netdata
+ExecStart=/usr/sbin/netdata -nd $EXTRA_OPTS
+TimeoutStopSec=30
+Restart=5s
```

Comment:
@shartge thanks - I didn't think it looked quite right, but then forgot to check. It works fine with this now. I'll push a new version with this in it soon.